Content.Server/Atmos/EntitySystems/AtmosphereSystem.HighPressureDelta.cs

```diff
@@ -79,7 +79,7 @@ private void AddMobMovedByPressure(MovedByPressureComponent component, PhysicsCo
 
             foreach (var fixture in fixtures.Fixtures.Values)
             {
-                _physics.RemoveCollisionMask(fixtures, fixture, (int) CollisionGroup.VaultImpassable);
```

Comment:
@metalgearsloth Do you know what this was for? If it was for getting atmosed over tables, then I need to change it to MidImpassable or better TableLayer.